### PR TITLE
For MRT Core, exclude file names and file paths from logging as they contain PII.

### PR DESCRIPTION
**Pull request by rohanp-msft**
I verified that the change builds fine.
There is an issue where none of the log data (HR values, etc.) are showing up in the ETL logs... Outside of that, I confirmed the events are logged. Proceeding with check-in. Will follow up on the issue separately.

### CODE REVIEW

File: dev/MRTCore/mrt/mrm/include/mrm/common/MrmTraceLogging.h [hunk 1]
```diff
@@ -31,22 +31,10 @@ class MrtRuntimeTraceLoggingProvider : public wil::TraceLoggingProvider
     DEFINE_COMPLIANT_MEASURES_EVENT_PARAM4(MeasureGenericEvent, PDT_ProductAndServicePerformance, PCWSTR, functionName, PCWSTR, message1, PCWSTR, message2, int, hresult);
 };
 
-// case insensitive prefix match of "*:\users"
-__inline PCWSTR RemoveUsernameFromPath(_In_ PCWSTR filePath)
+// Remove PII from string
+__inline PCWSTR RemovePiiFromString(PCWSTR)
```
Comment (DrusTheAxe) [inline dev/MRTCore/mrt/mrm/include/mrm/common/MrmTraceLogging.h:35]: `inline` ?

Comment (DrusTheAxe) [inline dev/MRTCore/mrt/mrm/include/mrm/common/MrmTraceLogging.h:35]: Rather generic name. `RemovePiiUserProfileFilename` would be more accurate

**Review by DrusTheAxe — APPROVED** [review]
Minor comments

**Review by jefgen — APPROVED** [review]
:shipit:

**Pull request merged**